helpers/AsyncFromSyncIterator.js

```diff
@@ -0,0 +1,230 @@
+'use strict';
+
+var internalSlot = require('internal-slot');
+var isCallable = require('is-callable');
+
+var callBind = require('./callBind');
+var callBound = require('./callBound');
+var GetIntrinsic = require('../GetIntrinsic');
+var $isNaN = require('./isNaN');
+
+var $apply = GetIntrinsic('%Reflect.apply%', true) || callBound('%Function.prototype.apply%');
+var $asyncIterator = GetIntrinsic('%Symbol.asyncIterator%', true);
+var $toStringTag = GetIntrinsic('%Symbol.toStringTag%', true);
+var $TypeError = GetIntrinsic('%TypeError%');
+
+var undefined;
+
+// TODO: Use %AsyncIterator.from% once it's in ECMA-262
+
+/** @type {(o: object, p: string | symbol, Desc: import('es-abstract').PropertyDescriptor) => boolean} */
+var DefineOwnProperty = callBind(
+	require('./DefineOwnProperty'),
+	undefined,
+	function IsDataDescriptor(Desc) {
+		return !('[[Get]]' in Desc) && !('[[Set]]' in Desc);
+	},
```

Comment:
this should use `require('./IsDataDescriptor')`.

Reply:
You do realise that this is now in the `helpers` directory, right?

As such, there is no `IsDataDescriptor` file that can be imported.

Reply:
In that case, it should take `IsDataDescriptor` as a function argument.